MonoGame.Framework.Content.Pipeline/Builder/PipelineBuildEvent.cs

```diff
@@ -176,7 +185,7 @@ public bool NeedsRebuild(PipelineManager manager, PipelineBuildEvent cachedEvent
 
             // If the source file is newer than the dest file
             // then it must have been updated and needs a rebuild.
-            if (sourceWriteTime >= destWriteTime)
+            if (sourceWriteTime > destWriteTime)
```

Comment:
If your output file is the source file copied to output, then the write times will exactly equal (and always will). Without this change, copy items always evaluate as needing a rebuild.